build-system/babel-plugins/babel-plugin-mangle-object-values/test/fixtures/transform/mangle/output.js

```diff
@@ -12,7 +12,217 @@ const y = notMangled({
 }); // mangled:
 
 const z = {
-  foo: "a",
-  bar: "b",
-  [x]: "c"
+  foo: 1,
+  bar: 2,
+  [x]: 3
+}; // dupes:
+
+const a = {
+  foo: 1,
+  baz: 2,
+  // should be same as foo:
+  bar: 1
+}; // 200 elements:
+
+const b = {
+  k0: 1,
+  k1: 2,
+  k2: 3,
+  k3: 4,
+  k4: 5,
+  k5: 6,
+  k6: 7,
+  k7: 8,
+  k8: 9,
+  k9: 10,
+  k10: 11,
+  k11: 12,
+  k12: 13,
+  k13: 14,
+  k14: 15,
+  k15: 16,
+  k16: 17,
+  k17: 18,
+  k18: 19,
+  k19: 20,
+  k20: 21,
+  k21: 22,
+  k22: 23,
+  k23: 24,
+  k24: 25,
+  k25: 26,
+  k26: 27,
+  k27: 28,
+  k28: 29,
+  k29: 30,
+  k30: 31,
+  k31: 32,
+  k32: 33,
+  k33: 34,
+  k34: 35,
+  k35: 36,
+  k36: 37,
+  k37: 38,
+  k38: 39,
+  k39: 40,
+  k40: 41,
+  k41: 42,
+  k42: 43,
+  k43: 44,
+  k44: 45,
+  k45: 46,
+  k46: 47,
+  k47: 48,
+  k48: 49,
+  k49: 50,
+  k50: 51,
+  k51: 52,
+  k52: 53,
+  k53: 54,
+  k54: 55,
+  k55: 56,
+  k56: 57,
+  k57: 58,
+  k58: 59,
+  k59: 60,
+  k60: 61,
+  k61: 62,
+  k62: 63,
+  k63: 64,
+  k64: 65,
+  k65: 66,
+  k66: 67,
+  k67: 68,
+  k68: 69,
+  k69: 70,
+  k70: 71,
+  k71: 72,
+  k72: 73,
+  k73: 74,
+  k74: 75,
+  k75: 76,
+  k76: 77,
+  k77: 78,
+  k78: 79,
+  k79: 80,
+  k80: 81,
+  k81: 82,
+  k82: 83,
+  k83: 84,
+  k84: 85,
+  k85: 86,
+  k86: 87,
+  k87: 88,
+  k88: 89,
+  k89: 90,
+  k90: 91,
+  k91: 92,
+  k92: 93,
+  k93: 94,
+  k94: 95,
+  k95: 96,
+  k96: 97,
+  k97: 98,
+  k98: 99,
+  k99: "a",
```

Comment:
If this is 3 chars, we could go up to 999.

Reply:
Might as well use numbers only. Any enum with ≥ 1000 elements would have other problems of its own 😬